prover/src/groth16_vk.rs

```diff
@@ -131,6 +131,7 @@ pub struct VKeyData {
     pub gamma_g2: String,
 }
 
+// TODO(Alin): This needs to be implemented more naturally via a TryFrom...
```

Comment:
it's unclear to me what the benefit of `TryFrom` is.

I only find it's harder to navigate to the function from callers in an IDE like clion: it takes you to the trait definition, which has thousands of `impl`s...

Making an explicit static function, instead, doesn't have this problem.

Reply:
I thought RustRover / CLion ask you for usages of the base trait declaration or of the actual implementation...

But, yes, that can be annoying.

But so can not being able to use `try_from` naturally.

IDK 🤷